src/main/java/istjbot/ui/Ui.java

```diff
@@ -0,0 +1,139 @@
+package istjbot.ui;
+
+import java.util.Scanner;
+
+/**
+ * Encapsulates the concept of text-based Ui representation of the bot.
+ * Responsible for printing out all user messages after the command is executed.
+ */
+public class Ui {
+    /** Scanner to read user input. */
+    private Scanner sc = new Scanner(System.in);
+
+    /**
+     * Prints out a line.
+     */
+    public void showLine() {
+        String line = "*_______________________________________________________________*";
+        System.out.println(line);
+    }
+
+    /**
+     * Prints out a welcome message.
+     */
+    public void showWelcome() {
+        this.showLine();
+        String welcomeMessage = "Hello! I'm IstjBot. \n" + "What can I do for you?";
+        System.out.println(welcomeMessage);
+        this.showLine();
+    }
+
+    /**
+     * Reads and returns the command user has typed for input.
+     *
+     * @return String of user input.
+     */
+    public String readCommand() {
+        String fullCommand = sc.nextLine();
+        return fullCommand;
+    }
+
+    /**
+     * Prints out a confirmation message that the task has been successfully added.
+     *
+     * @param tasksSize Number of tasks used whether to pluralize "task."
+     * @param addedTask String of the task added.
+     */
+    public void showTaskAdded(int tasksSize, String addedTask) {
+        String messageStart = "As an IstjBot, I will add this task right now. \n";
+        // May refactor messageLast?
+        String messageLast = "Now you have " + tasksSize + " ";
```

Comment:
Would 'initialMessage' and  'finalMessage' be better names for these 2 variables? :)